root/usr/share/nethesis/NethServer/Module/TlsPolicy.php

```diff
@@ -39,15 +39,16 @@ protected function initializeAttributes(\Nethgui\Module\ModuleAttributesInterfac
     public function initialize()
     {
         parent::initialize();
-        $this->declareParameter('Policy', $this->createValidator()->memberOf('', '20180330'), array('configuration', 'tls', 'policy'));
+        $this->declareParameter('Policy', $this->createValidator()->memberOf('', '20180330', '20180621'), array('configuration', 'tls', 'policy'));
     }
 
     public function prepareView(\Nethgui\View\ViewInterface $view)
     {
         parent::prepareView($view);
         $view['PolicyDatasource'] = array(
                 array('', $view->translate('Default_policy_label')),
-                array('20180330', $view->translate('Policy_item_label', array('2018-03-30'))),
+                array('20180330', $view->translate('policy_item_label', array('2018-03-30'))),
```

Comment:
Here `Policy_item_label` case changes: is it a typo?

Reply:
yes good catch

Reply:
good eyes (still, by now)